src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -206,6 +206,9 @@ public string HubName
         /// <summary>
         /// If UseAppLease is true, gets or sets the AppLeaaseOptions used for acquiring the lease to start the application.
         /// </summary>
+
+        public bool IncludeParameters { get; set; } = false;
```

Comment:
This change has inadvertently removed the documentation comments for the `AppLeaseOptions` property since you inserted your new method in between the existing method and its documentation comments. Please revert this change and instead declare your new property somewhere else, preferably below `AppLeaseOptions`.

Note that I can see the change in the generated documentation by looking at the changes to Microsoft.Azure.WebJobs.Extensions.DurableTask.xml, which is also included in this PR.

Also, please take some time to review this documentation which describes XML doc comments in C#: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/. Since your new `IncludeParameters` property is public, it will also require XML comments, just like all the other public properties declared in this file.

Reply:
Thanks for your reminder and help! I noticed it and have already changed it. Thanks for your help and @davidmrdavid. I already pushed the changed one here.